storage_service/locations/models/package.py

```diff
@@ -1220,7 +1224,7 @@ def finish_reingest(self, origin_location, origin_path, reingest_location, reing
         # If AIP is a directory, calculate size recursively
         if os.path.isdir(out_path):
             size = 0
-            for dirpath, _, filenames in os.walk(out_path):
+            for dirpath, dirs, filenames in os.walk(out_path):
```

Comment:
This was to prevent shadowing as `gettext` is imported as `_` now. Unlikely to cause actual problems with the tiny scope that it applies to, but linters like to complain. Alternatives would be to keep the linter quiet about it (I think it's pylint giving me this warning), or import gettext as something else